blog/2025-11-11-cloudstack-4.22-release/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Apache CloudStack 4.22.0.0 Release
+tags: [announcement]
+authors: [harikrishna]
+slug: cloudstack-4.22.0.0-release
+---
+
+[![](4.22.png "Apache CloudStack 4.22.0.0 Release")](/blog/cloudstack-4.22.0.0-release)
+
+The Apache CloudStack project is pleased to announce the release of CloudStack 4.22.0.0, which is a LTS release.
+
+The CloudStack 4.22 release is the most recent release of the cloud management platform. It comes as a product of extensive contributions from the project development community.
```

Comment:
```suggestion
Apache CloudStack 4.22.0.0 release is the most recent release of the cloud management platform and is a LTS release, guaranteeing ongoing maintenance and support for a period of 24 months. This release comes as a product of extensive contributions from the project development community.
```

@harikrishna-patnala Update the draft announcement also, inline with this. Thanks.